lib/framereader.py

```diff
@@ -174,6 +174,34 @@ def read_frame_P410(fd, width, height):
 
   return y, u, v
 
+def read_frame_Y410(fd, width, height):
+  size = width * height * 4
+  #https://docs.microsoft.com/en-us/windows/win32/medfound/10-bit-and-16-bit-yuv-video-formats
+  #y410 32bit
+  #U bit[9:0]
+  #Y bit[19:10]
+  #V bit[29:20]
+  #A bit[31:30]
+  y410 = numpy.fromfile(fd, dtype=numpy.uint8, count=size)
+
+  y410_dw_4 = y410[3::4].reshape((height, width))
+  y410_dw_3 = y410[2::4].reshape((height, width))
+  y410_dw_2 = y410[1::4].reshape((height, width))
+  y410_dw_1 = y410[0::4].reshape((height, width))
+
+  y410_high16 = (uint16_t) (y410_dw_4<<8 | y410_dw_3)
+  y410_low16 = (uint16_t) (y410_dw_2<<8 | y410_dw_1)
+  #bit 20-30
+  v= (uint16_t) ( (y410_high16&0x3ff0) >> 4 )
```

Comment:
It would be better to apply the same coding style, e.g. reserve a blank between operator and operand

Reply:
Good capture, Sure, I will

Reply:
updated patch